tests/unit_tests/stencil/core/test_functor_metafunctions.cpp

```diff
@@ -19,13 +19,13 @@ namespace gridtools {
                 using interval_t = interval<level<0, 1, 3>, level<2, -1, 3>>;
 
                 struct empty {};
-                //                static_assert(check_valid_apply_overloads<empty, interval_t>::value, "");
+                //                static_assert(check_valid_apply_overloads<empty, interval_t>::value);
```

Comment:
Why those commented assertions (also below)? Are those cases that _should_ work but actually don’t with the currently implemented algorithm?

Reply:
Explanation comment is added

Reply:
Thanks, that’s helpful indeed.